src/JuliaRunClient.jl

```diff
@@ -301,6 +305,17 @@ function _type_name_query(ctx::Context, path::String, job::JRunClientJob, query:
     parse_resp(resp)
 end
 
+function initializeCluster(num_workers, ctx=Context(), job=self())
```

Comment:
Since this API seems to be meant only to be used by a job to scale self, probably the parameter should be removed to avoid confusion.

Reply:
Ok, if it cannot be used in any other context, it makes sense to remove `job` param. I was not too sure. If so, let me know and I will make another PR.